src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -275,6 +279,21 @@ void FullyConnected::getSupportedDescriptors() {
 
     inDims = isDynamicNode() ? makeDummyInputDims() : getInputShapeAtPort(DATA_ID).getStaticDims();
     outDims = isDynamicNode() ? makeDummyOutputDims(inDims) : getOutputShapeAtPort(0).getStaticDims();
+#ifdef OV_CPU_WITH_MLAS
+    bool isINT8 = one_of(inputDataType, memory::data_type::u8, memory::data_type::s8) && weightsDataType == memory::data_type::s8;
+    // MLAS doesn't support post-ops fusing and BF16. INT8 is not enabled yet
```

Comment:
Have you tried to measure performance with disabled FC + post_ops fusion? For example common fusing from transformers is FC + Gelu activation. Given Gelu is quite compute intensive, fusing itself might be not benefitial at all. At the same time using mlas for the FC might bring much more perf improvements.

Reply:
Hi Dmitry, previous static shape/dynamic shape perf tests are testing with disabled FC + post_ops fusion. We see regression in some models. The reason here for disable MLAS when post-ops existing is to avoid regression. I think the tradeoff here is to balance normal models and LLM models

Reply:
Ok, thanks for sharing